src/Components/Scenes/SearchResults/SearchFilterChip/SearchFilterChip.js

```diff
@@ -19,21 +18,22 @@ const SearchFilterChip = props => {
   const chips = [];
   filterQueries.forEach((filteredChip, id) => {
     const filteredBy = Object.keys(filteredChip)[0];
-    let shortNamesforContain = filteredChip[filteredBy][1];
-    let contain;
-    if (shortNamesforContain === 'in') {
-      contain = 'contain';
-    } else if (shortNamesforContain === 'eq') {
-      contain = 'equal';
+    const filteredByCap = filteredBy.charAt(0).toUpperCase() + filteredBy.slice(1);
+    let shortNamesforType = filteredChip[filteredBy][1];
```

Comment:
Could make this a `const` for clarity, but that's also probably the most annoying nitpick I've ever written on github

Reply:
Fixed!